### ISSUE TITLE
run BERT-CRF, get loss is nan

### ISSUE THREAD

**Issue by yc1999** [THU-KEG/MAVEN-dataset#5]
Hi, thanks for your great work, these baselines help me understand the event detection task!  

When I run the code of BERT-CRF( I use 1 gpu, batch size = 16, Gradient Accumulation steps = 8), I get evaluation loss = nan, and p,r,f1-score = 0. 
![image](https://user-images.githubusercontent.com/37548571/132155563-232e602e-85f2-4059-952b-710302658364.png)

But when I turn the batch size to 2, it works fine.

So I don't know why batch size makes an impact on the result...😥could you help me figure out this confusion...

**Comment by wzq016** [comment]
Hi, thanks for your feedback!

I can't reproduce this error with one V100, batch size = 16, and Gradient Accumulation steps = 8.

During the first evaluation process, my p-r-f scores are zero and my loss is 45. 

I get normal p-r-f scores from the second eval process by the way.

Does this error only happen during your first evaluation process or all evaluation process?

**Comment by cnut1648** [comment]
Hi I can reproduce the same (loss = nan and p-r-f = 0) on the RTX 2080 Ti. I noticed that any experiment with batch size = 16 (no matter what the gradient accumulation is) would not work.

**Comment by cnut1648** [comment]
> Hi I can reproduce the same (loss = nan and p-r-f = 0) on the RTX 2080 Ti. I noticed that any experiment with batch size = 16 (no matter what the gradient accumulation is) would not work.
